netsim/augment/links.py

```diff
@@ -1048,8 +1048,7 @@ def set_default_gateway(link: Box, nodes: Box) -> None:
     if nodes[ifdata.node].get('role','') == 'host':       # Set gateway only for hosts
       for interface in nodes[ifdata.node].interfaces:     # Find the corresponding host interface
         if link.linkindex == interface.linkindex:
-          if interface.ifindex == 1:                      # Set the default gateway only on the first host interface
-            interface.gateway = link.gateway
+          interface.gateway = link.gateway                # Copy link gateway to host interface
```

Comment:
Should a "generic" host static routing framework depend on having gateway functionality enabled in the topology?

It is perfectly valid for a static route to have any router interface as next hop, gateway or no gateway. When present, the gateway makes for a natural candidate - but it should be possible to do static routing in topologies that do not use the gateway module